testing/trino-faulttolerant-tests/pom.xml

```diff
@@ -210,6 +210,11 @@
                     <groupId>org.apache.commons</groupId>
                     <artifactId>commons-lang3</artifactId>
                 </exclusion>
+                <exclusion>
+                    <!-- conflicts with com.google.protobuf:protobuf-java -->
+                    <groupId>org.apache.iceberg</groupId>
+                    <artifactId>iceberg-orc</artifactId>
```

Comment:
Excluding this looks like it can break the faulttolerant tests.

Why `iceberg-orc` conflicts with `com.google.protobuf:protobuf-java`?
is it some Iceberg packaging problem?

Reply:
I believe the issue is that `orc-core-nohive` ships google protobuf files and that conflicts. Alternatively we could probably just add `orc-core-nohive` to the ignored dependencies of `duplicate-finder-maven-plugin`

Reply:
Ignoring deps conflicts is not safe either. That's why duplicate finder plugin is being used.

Reply:
Can we fix `org.apache.orc:orc-core:jar:nohive:1.8.0:runtime` not to include protobuf stuff (or shade it correctly)?

Reply:
do you mean to fix this in Iceberg or in Trino?

Reply:
I think the problem is in Iceberg, and I don't see how we can fix this in Trino.